crates/stellar-xdr-wasm/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+mod utils;
+
+use std::{
+    cmp,
+    io::{self, Read},
+    str::FromStr,
+};
+
+use stellar_xdr::{
+    curr::{self, TypeVariant, WriteXdr},
+    schema::Schema,
+};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub fn type_variants() -> String {
+    format!(
+        "[{}]",
+        TypeVariant::VARIANTS_STR
+            .iter()
+            .map(|s| format!("{s:?}"))
+            .collect::<Vec<String>>()
+            .join(", ")
```

Comment:
We should remove the space so that the list is comma separated values that don't need to be trimmed. A UI / presentation layer can add in any spaces between values if it is presenting the values that way.
```suggestion
            .join(",")
```